<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CATS Chat</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
    </property>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::TabPosition::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::TabShape::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>-1</number>
      </property>
      <property name="elideMode">
       <enum>Qt::TextElideMode::ElideNone</enum>
      </property>
      <property name="usesScrollButtons">
       <bool>false</bool>
      </property>
      <property name="documentMode">
       <bool>true</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="msgBoxLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Fi&amp;le</string>
    </property>
    <widget class="QMenu" name="menuConfigure">
     <property name="title">
      <string>Configure</string>
     </property>
     <addaction name="configureRadio"/>
     <addaction name="configureStation"/>
    </widget>
    <addaction name="menuConfigure"/>
    <addaction name="fileExit"/>
   </widget>
   <widget class="QMenu" name="menuMessage">
    <property name="title">
     <string>Messa&amp;ge</string>
    </property>
    <addaction name="messageSendTo"/>
    <addaction name="messageClearAll"/>
   </widget>
   <widget class="QMenu" name="menuRadio">
    <property name="title">
     <string>Ra&amp;dio</string>
    </property>
    <addaction name="radioConnect"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuMessage"/>
   <addaction name="menuRadio"/>
  </widget>
  <action name="messageSendTo">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="messageClearAll">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Clear All</string>
   </property>
  </action>
  <action name="fileExit">
   <property name="text">
    <string>Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="fileSetup">
   <property name="text">
    <string>Setup</string>
   </property>
  </action>
  <action name="radioConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="configureRadio">
   <property name="text">
    <string>Radio</string>
   </property>
  </action>
  <action name="configureStation">
   <property name="text">
    <string>Station</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
